### ISSUE TITLE
Show actual width of roads

### ISSUE THREAD

**Issue by westnordost** [streetcomplete/streetcomplete-mapstyle#65]
Also in order to mitigate issues like https://github.com/westnordost/StreetComplete/issues/1250, it may be a good improvement to render the road in its actual width in meters on high zoom levels, if given (`width` key) or an approximated width if the `lanes` key is given. Currently, the road width is only approximated by the road type.

This feature is only possible if this information is actually available in the vector tile source. I did not look for it yet.

**Comment by waldyrious** [comment]
Not sure if this should be a separate issue, but perhaps a simpler solution/workaround could be to simply allow the roads to get thinner at high zoom levels.

This is quite often a problem for me, as the roads (and road segments) get all mashed together, making the actual geometry unclear. Example:

https://play.tangram.city/?#20.00000/41.54453/-8.43325

![screenshot_2019-03-07 tangram play](https://user-images.githubusercontent.com/478237/53954902-75300500-40cf-11e9-9ff4-f07f94c7cffc.png)

**Comment by matkoniecz** [comment]
From looking at https://tilezen.readthedocs.io/en/latest/layers/#road-transportation-kind_detail-values-and-zoom-ranges this data appears to not be available.

See https://github.com/tilezen/vector-datasource/issues/1021

**Comment by peternewman** [comment]
Is this not unblocked @westnordost now we've switched to JAWG?

**Comment by westnordost** [comment]
Why, does jawg provide the width?

Am 5. August 2020 05:13:12 MESZ schrieb Peter Newman <notifications@github.com>:
>Is this not unblocked @westnordost now we've switched to JAWG?

**Comment by peternewman** [comment]
> Why, does jawg provide the width?

:facepalm: I didn't even consider that, just thought it might be different now it's not Tilezen. I tried to have a quick look, but couldn't find obvious lists unless I'm looking in the wrong place.

**Comment by westnordost** [comment]
Yes there is no documentation, you'd need to toss it into maputnik to analyze the properties 

Am 5. August 2020 11:50:17 MESZ schrieb Peter Newman <notifications@github.com>:
>> Why, does jawg provide the width?
>
>:facepalm: I didn't even consider that, just thought it might be
>different now it's not Tilezen. I tried to have a quick look, but
>couldn't find obvious lists unless I'm looking in the wrong place.

**Comment by matkoniecz** [comment]
@westnordost recently deliberately redesigned width display to make roads narrower, in https://github.com/streetcomplete/streetcomplete-mapstyle/commit/15a2f792df7156008a3b83fdbdfb1820e658b638 and https://github.com/streetcomplete/streetcomplete-mapstyle/commit/b1e3ce8a44f38b005b6d5538b07ea9f38e9a87f0

Should it be treated as rejecting the idea presented here?

**Comment by westnordost** [comment]
Yes, correct

**Comment by waldyrious** [comment]
For the record, here's what the area mentioned [above](https://github.com/streetcomplete/streetcomplete-mapstyle/issues/65#issuecomment-470497556) looks like in the app now:

<img src="https://user-images.githubusercontent.com/478237/193254951-cd92f011-36f4-4d48-b6db-89788a6f2780.png" width="50%" />

IMO the issue with roads being squished together due to their width has been sufficiently addressed. Thanks!
